.circleci/config.yml

```diff
@@ -353,6 +361,17 @@ jobs:
             echo $SFDX_CI_DEVHUB_JWTKEY | base64 --decode > devhub.key
             sfdx force:auth:jwt:grant --clientid ${SFDX_CI_DEVHUB_CLIENTID} --username ${SFDX_CI_DEVHUB_USERNAME} --jwtkeyfile devhub.key --setdefaultdevhubusername --setalias devhub
             rm devhub.key
+      - run:
```

Comment:
Why do this instead of having a scheduled task that runs your tests every "x" amount of time e.g. run the tests once a day ?

Reply:
Thats a good question, I had not thought about a chron job for these tests.  Perhaps it would simplify the CI and be enough of a check.

One downside is a delay in knowing when something is broken and does that gap expose us to missing an issue before it released?

Reply:
One tricky part of the current script is that it will only run with workflows on the develop branch, we decided that is probably fine for now, keep it really simple. 
That wouldn't be an issue any more with a chron job, I would imagine we could just have it run on the main branch.

Reply:
Alrighty, after some discussion with the team, I'm going to switch this to a chron job.

Reply:
@lcampos @xyc Please see the `fix: run UI tests with chron job` commit to run Ui test once a day at midnight with the other jobs.